docker-compose.yaml

```diff
@@ -1,27 +1,32 @@
 name: 'metrics-drilldown'
 services:
   grafana:
-    container_name: 'grafana-gmd'
+    container_name: 'grafana'
     extends:
       file: .config/docker-compose-base.yaml
       service: grafana
     build:
       args:
         grafana_version: ${GRAFANA_VERSION:-11.6.0}
-    ports: !override
-      - '${GRAFANA_PORT:-3001}:3000'
+    ports:
+      - '${GRAFANA_PORT:-3001}:3000/tcp'
     environment:
       GF_SERVER_ROOT_URL: 'http://localhost:${GRAFANA_PORT:-3001}'
       GF_FEATURE_TOGGLES_ENABLE: exploreMetricsUseExternalAppPlugin
+      # prevents a Grafana startup error in case a newer plugin version (set in package.json) is used compared to the latest one published in the catalog
+      GF_PLUGINS_PREINSTALL_DISABLED: true
     volumes:
       - ./provisioning/dashboards/dashboardJson:/etc/dashboards
     extra_hosts:
       # This allows us to connect to other services running on the host machine.
       - 'host.docker.internal:host-gateway'
 
+  # We use this for displaying the metrics in our app
   prometheus:
-    container_name: 'prometheus-gmd'
-    image: prom/prometheus:v3.2.1
+    container_name: 'prometheus'
```

Comment:
Small nit, but I am happy to keep it this way, named "prometheus." I'm running another Tempo docker setup that contains prometheus and there was a conflict in names. 

Overall, LGTM though! 🚀

Reply:
oh, i can make another follow up PR for that